source/deployment/cluster.rst

```diff
@@ -0,0 +1,207 @@
+
+Adding a Server to the Cluster
+------------------------------------------------------------
+
+1. Backup your `Mattermost database <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_ and `file storage location <https://docs.mattermost.com/deployment/cluster.html#file-storage-configuration>`_.
+2. Setup a Mattermost server following one of our **Install Guides**. This server must use an identical copy of the configuration file, ``config.json``. Verify the server is functioning by hitting the private IP address.
+3. Modify the ``config.json`` files on all servers with the ``ClusterSettings`` as `described in this documentation <https://docs.mattermost.com/administration/config-settings.html#high-availability-beta>`_. Be sure to add the new server URL to ``InterNodeUrls``. 
+4. Verify the configuration files are identical on all servers, then restart each machine in the cluster in sequence with 10 or more seconds between each restart.
+5. `Modify your NGINX setup <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ to add the new server.
+6. Open the **System Console** > **Advanced** > **High Availability (Beta)** to verify all the machines in the cluster are communicating as expected with green status indicators. If not, investigate the log files for any extra information.
+
+Removing a Server from the Cluster
+-----------------------------------------------------------------
+
+1. Backup your `Mattermost database <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_ and `file storage location <https://docs.mattermost.com/deployment/cluster.html#file-storage-configuration>`_.
+2. `Modify your NGINX setup <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ to remove the server.
+3. On all servers staying active in the cluster, modify the ``ClusterSettings`` in ``config.json`` to remove the server from ``InterNodeUrls`` 
+4. Verify the configuration files are identical on all servers, then restart each machine in the cluster in sequence with 10 or more seconds between each restart.
+5. Open the **System Console** > **Advanced** > **High Availability (Beta)** to verify all the machines in the cluster are communicating as expected with green status indicators. If not, investigate the log files for any extra information.
+
+
+-----
+
+Configuration and Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~
+Details on configuring your system for high availability.    
+
+Mattermost Server Configuration
+------------------------------------------------
+
+Configuration Settings
+````````````````````````````````````
+High availability is configured in the ``ClusterSettings`` section of ``config.json`` and viewable in the System Console. When high availability is enabled, the System Console is set to read-only mode to ensure all the ``config.json`` files on the Mattermost servers are identical.
```

Comment:
maybe "and the settings are viewable in the System Console"? 0/5, wording just feels slightly awkward as is